packages/greedy_infomax/src/nupic/research/frameworks/greedy_infomax/models/full_model.py

```diff
@@ -52,8 +52,14 @@ class FullVisionModel(torch.nn.Module):
     A modified version of ResNet to compute patch-wise representations. This model
     is the encoder in self-supervised experiments and does not include a built in
```

Comment:
Are you still using these "full models" despite the more recent addition of specifying block models instead? It's important to know, since if you're not, then we can move this file to the archive folder.

Reply:
I don't expect them to be actively developed in the future, but moving it might break some code now. Let's touch base on this later